esbuild.js

```diff
@@ -20,7 +20,7 @@ const argv = minimist(process.argv.slice(2));
 // https://esbuild.github.io/getting-started/#build-scripts
 const buildConfig = {
   bundle: true,
-  minify: true,
+  minify: false,
```

Comment:
Does this mean all our outputs won't be minifed?

Reply:
opps, will fix, this was for testing.